src/hektor/electrons/parabolic/semimaxwellian/ne00p_phiinfty_nobarriers.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from scipy.special import erf
+from scipy.optimize import newton
+
+
+def ne00p_phiinfty_nobarriers(chi):
```

Comment:
We could write somewhere that `ne00p` corresponds to

![Screenshot_2020-07-27 Online LaTeX Equation Editor](https://user-images.githubusercontent.com/316517/88534717-f8296680-d008-11ea-8fc4-0b0de959be5e.png)

in the paper, and that `eq1_minus_eq2` corresponds to Eq. 42 (normalized in e and T_e^*)